src/Conversion/ONNXToTOSA/ONNXToTOSACommon.hpp

```diff
@@ -83,6 +83,33 @@ llvm::SmallVector<int64_t> getCeilConstants(llvm::ArrayRef<int64_t> inputShape,
   return llvm::SmallVector<int64_t>{xAxis, yAxis};
 }
 
+// Create an ArrayAttr of pad values from \p shapeHelper using \p padIndexOrder.
+// Values are calculated considering \p ceilMode.
+template <typename ShapeHelperType>
+mlir::ArrayAttr createOrderedPadValues(mlir::PatternRewriter &rewriter,
```

Comment:
The term "padding values" is making me thing of the values to put in the padding regions. Is that what this returns?

Reply:
`Padding values` was the nomenclature previously used in the pass to refer to the actual amount of padding to use. In `PaddingOp`, its adaptor uses `pads()` for it, and `constant_value()` to denote the padding filling value to use. 
TBH, I am also thinking that we could remove the term `value` and just leave it as pad?

Reply:
Changed to `createOrderedPadAttr`